.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Question
+about: Do you have question regarding PMDK? Please let us know.
+
+---
+# QUESTION: <!-- fill the title of question -->
+
+## Details
+
+<!-- fill this out -->
+
+For questions and other non-bugs, You could use http://groups.google.com/group/pmem
```

Comment:
Did you mean to put this note in comment (\<!-- -->)?

Reply:
I added this in separate comment, if You prefer this comment to be in one section let me know